packages/cli/src/executable/AgenticaStart.ts

```diff
@@ -0,0 +1,63 @@
+import inquirer from "inquirer";
+import fs from "fs";
+import path from "path";
+import { getNpmPackages } from "../utils/getNpmPackages";
+import { getQuestions } from "../utils/getQuestions";
+import { createProjectDirectory } from "../utils/createProjectDirectory";
+import { Connector } from "../bases/Connector";
+import { Package } from "../bases/Package";
+import { Tsconfig } from "../bases/Tsconfig";
+
+export namespace AgenticaStart {
+  export async function execute(input: { projectName: string }) {
+    const { projectName } = input;
+
+    // Get connector package names from npm and sort alphabetically
+    const availableServices = (await getNpmPackages()).sort((a, b) =>
+      a.name.localeCompare(b.name)
+    );
+
+    const questions = getQuestions({ services: availableServices });
+    const answers = await inquirer.prompt(questions as any);
```

Comment:
```suggestion
    const answers = await inquirer.prompt(questions);
```
please remove unnecessary any. It only adds confusion to the code.